src/main/java/uk/gov/hmcts/divorce/document/DocumentGenerator.java

```diff
@@ -30,6 +30,23 @@ public class DocumentGenerator {
     private final CaseDataDocumentService caseDataDocumentService;
     private final Clock clock;
 
+    public void generateAndStoreCaseDocument(DocumentType documentType,
+                                             String templateId,
+                                             String templateName,
+                                             CaseData caseData,
+                                             long caseId) {
+        //this is a case document like FO Granted or CO Granted so is not specific to an applicant
+        var templateContent = getTemplateContent(caseId, null, caseData, documentType, templateId);
+
+        var generatedDocument = caseDataDocumentService.renderDocument(templateContent,
+                caseId,
+                templateId,
+                caseData.getApplicant1().getLanguagePreference(),
```

Comment:
Is hardcoding to applicant1 going to cause issues?

Reply:
Discussed with Linus and it seems like language preference is always selected based on applicant1